### PR TITLE
flutter: add missing features in the features page

### PR DESCRIPTION
**Pull request by buenaflor**
## DESCRIBE YOUR PR
Flutter Features page has some missing information

Closes https://github.com/getsentry/sentry-dart/issues/2117

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: docs/platforms/dart/features/index.mdx [hunk 1]
```diff
@@ -20,3 +20,4 @@ Sentry's Dart SDK enables automatic reporting of errors and exceptions, and iden
   - [File I/O Integration](/platforms/dart/integrations/file/).
 - [Logging Integration](/platforms/dart/integrations/logging).
 - [Dio Integration](/platforms/dart/integrations/dio/).
+- [Metrics](/product/performance/metrics/) help you track custom data points to gain insights into app performance and user experience
```
Comment (lizokm) [inline docs/platforms/dart/features/index.mdx:23]:
```suggestion
- [Metrics](/product/performance/metrics/) help you track custom data points to gain insights into app performance and user experience.
```
Hey there, just want to double check that you want this to point to performance metrics not these [metrics](https://docs.sentry.io/product/explore/metrics/)...

File: docs/platforms/flutter/features/index.mdx [hunk 2]
```diff
@@ -22,4 +22,8 @@ Sentry's Flutter SDK enables automatic reporting of errors and exceptions, and i
     - as well as `http` with the [Dart SDK](/platforms/dart/).
 - Integrations for sqflite, routing and more. For a complete list, see our [integrations list](/platforms/flutter/integrations/).
 - [Release Health](/product/releases/health/) tracks crash free users and sessions.
-- Limited support for Flutter Web.
+- [Attachments](/platforms/flutter/enriching-events/attachments/) that can enrich your event by storing additional files, such as config or log files.
+- [User Feedback](/platforms/flutter/user-feedback/), providing the ability to collect user information when an event occurs.
```
Comment (lizokm) [inline docs/platforms/flutter/features/index.mdx:26]:
```suggestion
- [User Feedback](/platforms/flutter/user-feedback/), providing the ability to collect user feedback when an unexpected event occurs.
```

**Review by lizokm — APPROVED** [review]
Thanks for adding! Just a few small tweaks, but this looks good.

File: docs/platforms/dart/features/index.mdx [hunk 3]
```diff
@@ -20,3 +20,4 @@ Sentry's Dart SDK enables automatic reporting of errors and exceptions, and iden
   - [File I/O Integration](/platforms/dart/integrations/file/).
 - [Logging Integration](/platforms/dart/integrations/logging).
 - [Dio Integration](/platforms/dart/integrations/dio/).
+- [Metrics](/product/performance/metrics/) help you track custom data points to gain insights into app performance and user experience.
```
Comment (kahest) [inline docs/platforms/dart/features/index.mdx:23]: I'd not add Metrics now, seeing as it's very likely we'll remove or drastically change the API soon

**Review by kahest — APPROVED** [review]

**Pull request merged**